spec/schemas/indices._common.yaml

```
@@ -1155,10 +1303,12 @@ components:
           $ref: '_common.yaml#/components/schemas/ByteCount'
         indices:
           type: object
+          description: The upgrade status for individual indices.
```

Per Vale lint, this should be:
```suggestion
          description: The upgrade status for individual indexes.
```